File: composeApp/src/desktopMain/java/ies/java/model/Provincia.java
package ies.java.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "provincia")
public class Provincia {
    @Id
    @Column(name = "codp", nullable = false, length = 2)
    private String codp;

    @Column(name = "nombre", nullable = false, length = 25)
    private String nombre;

    public String getCodp() {
        return codp;
    }

    public void setCodp(String codp) {
        this.codp = codp;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

}

File: composeApp/src/desktopMain/java/ies/java/model/Provincia.kt
package ies.java.model

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "provincia")
open class Provincia {
    @Id
    @Column(name = "codp", nullable = false, length = 2)
    open var codp: String? = null

    @Column(name = "nombre", nullable = false, length = 25)
    open var nombre: String? = null
}